### PR TITLE
make bs crystals easier to get

### PR DESCRIPTION
**Pull request by lanse12**
## About the PR
made bs crystals way easier to get
they spawn more often and drop more when mined
also obliterated salt from asteroids because it's fucking useless
i think i also made a few ores less rare

## Why / Balance
some friend removed them from protolathe so i'm giving them a little bit of love

**Changelog**
:cl:
- tweak: BS crystals are now way easier to obtain.

### CODE REVIEW

**Comment by BombasterDS2** [comment]
Don't remove salt - it's main purpose to be sold.
With BSCrystal - it works. Anyway, mining now needs rebalance (after previous rebalance)

**Comment by lanse12** [comment]
nobody fucking sells salt dude it's uselses

File: Resources/Prototypes/Entities/World/Debris/asteroids.yml [hunk 1]
```diff
@@ -27,13 +27,13 @@
             prob: 0.15
             orGroup: rock
           - id: WallRockSalt
-            prob: 0.15
+            prob: 0.0001
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/World/Debris/asteroids.yml]: just remove it literally nobody likes salt

Comment (SolsticeOfTheWinter) [reply Resources/Prototypes/Entities/World/Debris/asteroids.yml:30]: W, fuck salt

**Pull request merged**